openfe/protocols/openmm_rbfe/__init__.py

```diff
@@ -5,7 +5,10 @@
 
 from .equil_rbfe_methods import (
     RelativeLigandProtocol,
-    RelativeLigandProtocolSettings,
     RelativeLigandProtocolResult,
     RelativeLigandProtocolUnit,
 )
+
+from .equil_rbfe_settings import (
```

Comment:
since Protocol depends on Settings, would it make more sense to import this first?

Reply:
do you mean moving it up the import list? sure

Reply:
yes, it looks like a circular import?  I guess they don't exist any more